src/js/tests/strings.rs

```diff
@@ -0,0 +1,16 @@
+use super::super::*;
+use crate::assert_js;
```

Comment:
I'm not sure exposing the macro on the crate is the neatest. But I was struggling to access the macro from a sub module and this was a solution that works.